.changes/nextrelease/changelog_auto_assume_role.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "type": "enhancement",
```

Comment:
The scope of this change qualifies as a `feature` and should get the minor version bump.